Gemfile

```diff
@@ -3,8 +3,10 @@
 source 'https://rubygems.org'
 ruby IO.read(File.expand_path('.ruby-version', __dir__)).strip
 
+# autoprefixer-rails 10 or greater requires node.js 12 or greater
+gem 'autoprefixer-rails', '~> 9'
 gem 'bootstrap', '~> 4.3.1'
-gem 'bootstrap_form'
+gem 'bootstrap_form', '~> 4.3.0'
```

Comment:
`bootstrap_form >= 4.5` depends on `autoprefixer-rails >= 10`. `autoprefixer-rails >= 10` requires `node.js >= 12`. Currently the server is running `node.js 11.15.0`.

Reply:
So, I don't, um, see this to be the case. There's no mention of that requirement in `bootstrap_form`'s README or gemspec.

Also — minor correction — our server is running node.js 10.15.2